es6/es6.tag

```diff
@@ -0,0 +1,10 @@
+<es-6>
+
+  <p>{ message }</p>
+
+  <script type="es6">
+    const name = 'Riot'
+    this.message = `Hello ${name}`
```

Comment:
I'll add an example method and some text in the readme saying something along the lines of "using `type="es6|typescript|coffeescript"` will override Riot's built in pre-processing" and keep the link to the docs. Maybe we need a similar line in the docs?